packages/flutter/lib/src/cupertino/text_theme.dart

```diff
@@ -80,7 +80,7 @@ const TextStyle _kDefaultPickerTextStyle = TextStyle(
   fontFamily: '.SF Pro Display',
   fontSize: 21.0,
   fontWeight: FontWeight.w400,
-  letterSpacing: -0.41,
+  letterSpacing: -0.6,
```

Comment:
Can you add a description on how you derived this value? This will help the next maintainer assess his confidence on whether that person is more rigorous or less than you are :)

e.g. extracted from Apple sketch specs or compared visually etc.

Reply:
To do this, I've made two comparison images that overlap the native text style version and the flutter text new and old style version, respectively.

This is a comparison of native and old version: 
![old](https://user-images.githubusercontent.com/11473691/92782435-8bf79c00-f3d7-11ea-9f0b-84632b9fa179.png)

This is a comparison of native and new version:
![new](https://user-images.githubusercontent.com/11473691/92782556-a0d42f80-f3d7-11ea-9072-ba98aaa681f7.png)

It doesn't look like much of a difference, but the new style version will be a little more accurate.

Reply:
Sounds good, I meant could you modify the code comment above this const declaration? It'll leave some hints for the next maintainer.

Reply:
Maybe I'll add a comment:

```
// LetterSpacing sourced from iOS 14 simulator screenshots for comparison.
See also:
//
// * https://github.com/flutter/flutter/pull/65501#discussion_r486557093
```

What do you think?

Reply:
For code comments, it's ok. (I wouldn't reference /// docs to ephemeral discussions like GitHub PRs).

Reply:
It's updated now.